python/langsmith/client.py

```diff
@@ -829,6 +845,7 @@ def request_with_retries(
                         emsg = msg
                     raise ls_utils.LangSmithError(
                         f"Failed to {method} {pathname} in LangSmith API. {emsg}"
+                        f"{_context}"
```

Comment:
Should we gate this behind some kind of flag to prevent unintended leakage into logs in error cases?  Thinking about whether there'd be PII in here (potentially?) and that we'd maybe want someone to set an easy flag to surface this?  I might be barking up wrong tree here.

Reply:
Hm we could, though I think I'd want it to be opt-out since we're only returning run/trace IDs for the context?

Reply:
Ah OK, this is not the full payload.  Maybe *that* (pretty printed) is what we'd (eventually) want to add in as an opt-in.

Reply:
Ya that could be an interesting opt-in bit. Right now I'm just doing the IDs so we can correlate them; anything else you'd want to see by default?